typegen.json

```diff
@@ -37,12 +37,10 @@
     "Battlepass.BattlepassActivated",
     "Battlepass.BattlepassEnded",
 
-    "Uniques.Created",
-    "Uniques.Issued",
-    "Uniques.CollectionMetadataSet",
-    "Uniques.MetadataSet",
-    "Uniques.CollectionMaxSupplySet",
```

Comment:
What about handling `CollectionMaxSupplySet` and `Destroyed` events? I can see they exist in the pallet but their support is dropped here. How can we handle if the collection is removed?

Reply:
In pallet_nfts `max supply` is set when a collection is created, there is no need to call separate extrinsic for it, hence it doesn't send the `CollectionMaxSupplySet` event.
`Destroyed` handler I think we added only for testing purposes. I don't see why we need it now.

Reply:
destroyed you need imho to remove void items from the index

Reply:
Yes, it helped when we had some issues with `collection_id` inconsistency. 
pallet_nfts increments `collection_id` internally, so we don't need this workaround anymore.

Reply:
As for the `CollectionMaxSupplySet` - apart from setting max supply while creating a collection, it is also possible to change this value with dedicated extrinsic, which generates event. So there is inconsistency in processing of max supply value.
In general, I don't see a reason to remove existing handlers, especially those which can cause change in the storage of NFT objects we need to keep track of. So it would be good to have those processors in place.

Reply:
sudo can call filtered out calls, but this should be fine for now.